wmn-data.json

```diff
@@ -2004,6 +2004,16 @@
         "known" : ["tommy", "cderalow"],
         "cat" : "health"
        },
+        {
+        "name" : "GeeksForGeeks",
+        "uri_check" : "https://www.geeksforgeeks.org/user/{account}",

```

Comment:
When I visit this URL it redirects me to `https://www.geeksforgeeks.org/user/{account}/`. Please add the trailing `/`.